frontend/src/cljs/imas_seamap/views.cljs

```diff
@@ -692,6 +692,9 @@
       (keydown-wrapper
        {:label "Toggle Sidebar"         :combo "s"}
        [:ui.sidebar/toggle])
+      (keydown-wrapper
+       {:label "Toggle Seamap Drawer"   :combo "a"} ;; TODO: Better label and perhaps different key mapping?
```

Comment:
Since you raised it -- "seamap" probably doesn't need to be included :)  ("d"is available, but I suspect we'll overhaul these with the new UI anyway so I wouldn't waste any time over it)